test/functional/test-viewport-binding.js

```diff
@@ -438,293 +438,292 @@ describes.realWin('ViewportBindingIosEmbedWrapper', {ampCss: true}, env => {
   });
 });
 
-describes.realWin('ViewportBindingIosEmbedShadowRoot_', {ampCss: true}, env => {
+describes.realWin('ViewportBindingIos', {ampCss: true}, env => {
```

Comment:
Please use the old name. We have multiple iOS bindings.

Reply:
Reverted.